src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/http_action/config/PerformanceAnalyzerResourceProviderTests.java

```diff
@@ -43,7 +43,7 @@
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({PluginSettings.class})
 @SuppressStaticInitializationFor({"PluginSettings"})
-public class PerformanceAnalyzerResourceProviderTest {
+public class PerformanceAnalyzerResourceProviderTests {
```

Comment:
Why was this rename needed?

Reply:
To run these UTs while building. In out gradle setup we filter test files via `*Tests` Suffix. These were not running till now.